### ISSUE TITLE
Not available in Africa (Cape Town) Region

### ISSUE THREAD

**Issue by jojo786** [rpidanny/gm-lambda-layer#11]
Trying to add arn:aws:lambda:af-south-1:175033217214:layer:graphicsmagick:2 results:

Resource handler returned message: "User:             
                                                                                                                                                                     arn:aws:iam::111111111:user/myuser is not        
                                                                                                                                                                        authorized to perform: lambda:GetLayerVersion on      
                                                                                                                                                                        resource: arn:aws:lambda:af-                          
                                                                                                                                                                        south-1:175033217214:layer:graphicsmagick:2 because   
                                                                                                                                                                        no resource-based policy allows the                   
                                                                                                                                                                        lambda:GetLayerVersion action (Service: Lambda,       
                                                                                                                                                                        Status Code: 403, Request ID:                         
                                                                                                                                                                        c167edda-b3dd-4f16-8410-fdd959f0f6cb, Extended        
                                                                                                                                                                        Request ID: null)" (RequestToken:                     
                                                                                                                                                                        b2b3fbab-0091-5ebf-f2ed-b9fbff6c7de3,                 
                                                                                                                                                                        HandlerErrorCode: AccessDenied)

**Comment by michaelwiles** [comment]
I don't think that error has anything to do with it "not being available in Africa"... I suspect your user just doesn't have the requisite permissions.

**Comment by michaelwiles** [comment]
Confirmed - I just successfully uploaded the layer to af-south-1

**Comment by jojo786** [comment]
My user has AdministratorAccess
If I try from the aws cli:
aws lambda get-layer-version --layer-name arn:aws:lambda:af-south-1:175033217214:layer:graphicsmagick --version-number 2 --region af-south-1

An error occurred (AccessDeniedException) when calling the GetLayerVersion operation: User: arn:aws:iam::718974227478:user/jojo786 is not authorized to perform: lambda:GetLayerVersion on resource: arn:aws:lambda:af-south-1:175033217214:layer:graphicsmagick:2 because no resource-based policy allows the lambda:GetLayerVersion action

Tried with versions 1 and 3 as well.

But using us-east-1 (or other regions except af-south-1), with version 1 and 2, works as expected:
aws lambda get-layer-version --layer-name arn:aws:lambda:us-east-1:175033217214:layer:graphicsmagick --version-number 2 --region us-east-1
